corehq/apps/locations/dbaccessors.py

```diff
@@ -92,3 +92,23 @@ def user_ids_at_accessible_locations(domain_name, user):
     from corehq.apps.locations.models import SQLLocation
     accessible_location_ids = SQLLocation.active_objects.accessible_location_ids(domain_name, user)
     return user_ids_at_locations(accessible_location_ids)
+
+
+def assigned_location_by_user_id(domain, location_ids):
```

Comment:
`location` filters on the assigned locations, not primary location: https://github.com/dimagi/commcare-hq/blob/je/ls-indicator-num-2/corehq/apps/es/users.py#L112-L131

I was trying to not rely too much on icds's current structure and it seemed like this would be equivalent. I can write a separate dbaccessor for using primary location id though